userspace/falco/configuration.h

```diff
@@ -37,6 +37,9 @@ limitations under the License.
 #include "event_drops.h"
 #include "falco_outputs.h"
 
+// Falco only metric
+#define METRICS_V2_JEMALLOC_STATS 1 << 31
```

Comment:
All other `METRICS_V2_...` defines live in libs; give this a big number to avoid conflicts.